benchmarks/Kestrel.Performance/Kestrel.Performance.csproj

```diff
@@ -4,7 +4,7 @@
     <!-- Using shorter assembly name instead of Microsoft.AspNetCore.Server.Kestrel.Performance because https://github.com/dotnet/BenchmarkDotNet/issues/498 -->
     <AssemblyName>Kestrel.Performance</AssemblyName>
     <RootNamespace>Microsoft.AspNetCore.Server.Kestrel.Performance</RootNamespace>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
```

Comment:
Didn't we want to keep these at netcoreapp2.1 on the release/2.2 branch? https://github.com/aspnet/KestrelHttpServer/pull/3014#discussion_r225269003

Reply:
I misinterpreted that comment.